pkg/router/template/router.go

```diff
@@ -301,6 +302,21 @@ func genCertificateHostName(hostname string, wildcard bool) string {
 	return hostname
 }
 
+// Returns the list of endpoints for the given route's service
+// action argument further processes the list e.g. shuffle
+// The default action is in-order traversal of internal data structure that stores
+//   the endpoints (does not change the return order if the data structure did not mutate)
+func processEndpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit, action string) []Endpoint {
+	endpoints := endpointsForAlias(alias, svc)
+	if strings.ToLower(action) == "shuffle" {
```

Comment:
Any particular reason for "shuffle" over "random" or something like that?  Also are there docs somewhere that describe how you use ROUTER_BACKEND_PROCESS_ENDPOINTS, or is that just a normal pod definition environment variable?